frontend/src/components/EditProfile/EditProfile.js

```diff
@@ -0,0 +1,147 @@
+                    return <Redirect to="/register"></Redirect>
+                })
+        }
+        fetchCurrentUser();
+    }, [loading])
+
+    useEffect(() => {
+        //This useEffect purpose is to force the page to re-render without having to make an API request
+    }, [displayName, biography, email, editStatus])
+
+    const handleTyping = (e, setterFunction) => {
+        setterFunction(e.target.value);
+    }
+
+    const handleEditStatus = (e, status) => {
+        e.preventDefault();
+        setEditStatus(status);
+    }
+
+    const updateProfile = (event) => {
+        event.preventDefault();
+        Axios.put(global.config.BACKEND_URL + `/account/profile/biographyUpdate`,
+            {
+                "biography": event.target.biography.value
+            });
+
+        Axios.put(global.config.BACKEND_URL + `/account/profile/emailUpdate`,
+            {
+                "email": event.target.email.value
+            }).catch((error) => {
+                alert(error.response.data.message);
+            })
+
+        Axios.put(global.config.BACKEND_URL + `/account/profile/displayNameUpdate`,
+            {
+                "displayName": event.target.displayName.value
+            }).catch((error) => {
+                alert(error.response.data.message);
+            });
+    }
+    const updateAccount = (event) => {
+        event.preventDefault();
+        Axios.put(global.config.BACKEND_URL + `/account/passwordModification`,
+            {
+                "password": event.target.password.value
+            }).catch((error) => {
+                alert(error.response.data.message);
+            });
+
+        Axios.put(global.config.BACKEND_URL + `/account/birthdayModification`,
+            {
+                "birthday": event.target.birthday.value
+            }).catch((error) => {
+                alert(error.response.data.message);
+            })
+    }
+
+    return (
+        <div>
+            {
+                loading ? 'loading' :
+                    <div className="edit-profile-page">
+                        <div className="sidebar">
+                            <nav class="nav">
+                                <ul>
+                                    <li class="active"><a href="#!">Instagram++</a></li>
+                                    <li><a href="#!" onClick={(e) => handleEditStatus(e, "profile")}>Edit Profile</a></li>
+                                    <li><a href="#!" onClick={(e) => handleEditStatus(e, "account")} >Edit Account</a></li>
+                                    <li><a href="#!">Contact Support</a></li>
+                                </ul>
+                            </nav>
+                        </div>
+                        {
+                            (editStatus === "profile") ?
+
+                                <div className="edit-profile">
+                                    <form className="edit-profile-form" onSubmit={updateProfile}>
```

Comment:
As discussed offline, instead of having a form nested in a div, you can simply keep all of it contained within the `edit-profile` div and move your function call to your button wherein `onClick={updateProfile}`